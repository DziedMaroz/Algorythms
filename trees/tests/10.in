10
7
9
8
12
11
13
5
6
4
