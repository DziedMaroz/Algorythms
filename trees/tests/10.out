10
7
5
4
6
9
8
12
11
13
